Jenkinsfile.integration

```diff
@@ -23,12 +26,7 @@ pipeline {
     }
 
     environment {
-        // ROOKCHECK_CLUSTER_PREFIX = "${cluster_prefix}"
-        // FIXME(jhesketh): ${cluster_prefix} is too long for a node name.
```

Comment:
I think we need to leave this warning in here (or at least somewhere) until we have a chance to confirm the max length of a node name (and therefore the cluster prefix)

Reply:
That makes sense to me.

Reply:
> Thanks for the changes! :-)
> 
> We should consider getting rid of configuration.env altogether as it is somewhat superseded by the examples folder. Nevertheless these fixes are good for now.
> 
Yeah... and rewrite the Quickstart guidelines.

> I'm not sure why Jenkins is reporting twice on this. I wonder if it is to do with the change to the JenkinsFile? Lets see if we get a good build after reworking the nit picks.

After I added the Jenkinsfile changes I noticed that jenkins does not pick this changes up, and the pipeline is constructed from older version, I have restarted the job manually to reproduce. Later I noticed that the job grabs the head branch instead of merge, and tried to change this in config, however it is failed unexpectedly. I still don't know the reason for sure, either the master branch has issues (however I rebased my branch to master) or the pipeline/github plugin merged it incorrectly.
I am about to drop this commit from the PR.

> 
> (Also, FYI, the ROOKCHECK_LIBVIRT_IMAGE can be a path to the image on your local machine which is why the example was as it was).

Yeah I know, but the issue that manual does not say a word that you need to download any image, and where this image should be placed, in your current directory or to some storage. The direct URL looks fine to me, in particular, since this is a cloud image for the guest system, and should not cause a problem for people who runs rookcheck on different distros.

Reply:
> > Thanks for the changes! :-)
> > We should consider getting rid of configuration.env altogether as it is somewhat superseded by the examples folder. Nevertheless these fixes are good for now.
> 
> Yeah... and rewrite the Quickstart guidelines.
> 
> > I'm not sure why Jenkins is reporting twice on this. I wonder if it is to do with the change to the JenkinsFile? Lets see if we get a good build after reworking the nit picks.
> 
> After I added the Jenkinsfile changes I noticed that jenkins does not pick this changes up, and the pipeline is constructed from older version, I have restarted the job manually to reproduce. Later I noticed that the job grabs the head branch instead of merge, and tried to change this in config, however it is failed unexpectedly. I still don't know the reason for sure, either the master branch has issues (however I rebased my branch to master) or the pipeline/github plugin merged it incorrectly.
> I am about to drop this commit from the PR.

That's odd. I'm working on another PR that modifies the jenkinsfile and it is picked up just fine. Lets see what happens after it is dropped.

> 
> > (Also, FYI, the ROOKCHECK_LIBVIRT_IMAGE can be a path to the image on your local machine which is why the example was as it was).
> 
> Yeah I know, but the issue that manual does not say a word that you need to download any image, and where this image should be placed, in your current directory or to some storage. The direct URL looks fine to me, in particular, since this is a cloud image for the guest system, and should not cause a problem for people who runs rookcheck on different distros.

:+1: